packaging/ubuntu-14.04/rules

```diff
@@ -122,7 +122,10 @@ override_dh_auto_build:
 	# Build golang bits
 	mkdir -p _build/src/$(DH_GOPKG)/cmd/snap/test-data
 	cp -a cmd/snap/test-data/*.gpg _build/src/$(DH_GOPKG)/cmd/snap/test-data/
-	dh_auto_build -- $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS)
+	dh_auto_build -- $(BUILDFLAGS) -tags "$(TAGS)" $(GCCGOFLAGS)
```

Comment:
Will tihs fail if "$(TAGS") is empty? Looking at it naively it seems like it might?

Reply:
Ah, indeed /o\, thanks for spotting!